metrics/ci_integrations/test/client/github_actions/mappers/run_conclusion_mapper_test.dart

```diff
@@ -0,0 +1,197 @@
+import 'package:ci_integration/client/github_actions/mappers/run_conclusion_mapper.dart';
+import 'package:ci_integration/client/github_actions/models/run_conclusion.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group("RunConclusionMapper", () {
+    const runConclusionMapper = RunConclusionMapper();
```

Comment:
```suggestion
    final mapper = RunConclusionMapper();
```
As this class is under tests we should avoid const constructors as we do on the web project.